src/utility/HCI.cpp

```diff
@@ -112,6 +112,15 @@ void HCIClass::poll(unsigned long timeout)
 
   while (HCITransport.available()) {
     byte b = HCITransport.read();
+	
+	if (sizeof(_recvBuffer) <= _recvIndex)
+	{
+	  _recvIndex = 0;
+	  if (_debug) {
+			_debug->println("_recvBuffer overflow");
+		  }
+	  continue;
```

Comment:
Why this statement here? Since you are resetting `_recvIndex` to zero you can write to `_recvBuffer` without any overflow.

Reply:
Yes, you are right. No need to add this code here.